plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -263,20 +263,18 @@ public static void main(String[] args)
         }
     }
 
-    public static class DeltaLakeGlueQueryRunnerMain
+    public static class DeltaLakeExternalQueryRunnerMain
     {
         public static void main(String[] args)
                 throws Exception
         {
-            // Requires AWS credentials, which can be provided any way supported by the DefaultProviderChain
-            // See https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html#credentials-default
-            DistributedQueryRunner queryRunner = builder(createSession())
+            // Please set Delta Lake connector properties via VM options. e.g. -Dhive.metastore.uri=thrift://localhost:9083 -D..
```

Comment:
the comment says to set `hive.metastore.uri`. But for running Delta query runner with Glue metastore requires setting `hive.metastore`.

How about if we can have `DeltaLakeGlueQueryRunnerMain` and `DeltaLakeExternalQueryRunnerMain` both?

Reply:
Can we update the comment then to include `hive.metastore`?